site/content/blog/2024-12-02-zap-updates-november-2024/index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "ZAP Updates - November 2024"
+summary: >
+  A brand new Scan Policies add-on, how to integrate ZAP with OWASP Noir and ZAP 2.16.0 is getting very close..
+images:
+- https://www.zaproxy.org/blog/2024-12-02-zap-updates-november-2024/images/zapbot-monthly-updates.png
+type: post
+tags:
+- blog
+- update
+date: "2024-12-02"
+authors:
+- psiinon
+---
+
+## Highlights
+
+### Scan Policies Add-On
+
+This month we released a new [Scan Policies](/docs/desktop/addons/scan-policies/) add-on which provides a set of
+Scan Policies tuned for different purposes.
```

Comment:
...purposes and audiences.

or "personas" ??

Reply:
Copied directly from https://www.zaproxy.org/docs/desktop/addons/scan-policies/ 😁

Reply:
Okay, not worth blocking on.